broker-core/src/main/java/io/zeebe/broker/system/configuration/ThreadsCfg.java

```diff
@@ -19,7 +19,7 @@
 
 public class ThreadsCfg implements ConfigurationEntry {
   private int cpuThreadCount = 2;
-  private int ioThreadCount = 2;
+  private int ioThreadCount = 0;
```

Comment:
Remove this settings completely from the configuration please, as we do not use any io threads anymore.